scripts/retrieveTranslations.js

```diff
@@ -0,0 +1,116 @@
+const fs = require('fs');
+const util = require('util');
+const {CimpressTranslationsClient} = require("cimpress-translations");
+const KmsAuthenticator = require("./kmsClientIdAuthorizer");
+const iso639 = require('iso-639').iso_639_2;
```

Comment:
This is a bit odd... but https://translations.cimpress.io works only with 3-letter language code which is inconvenient since lots of packages work with the 2-digit ones.

Reply:
Yes, we should definitely provide this feedback, it would be easier if they provide both so people can decide how to handle it. I'd like to avoid adding yet another package just to be able to find the 2digit equivalent. (It was really odd too see "spa" instead of "es"... or "bul" instead of "bg")

Storing both versions 2/3 digits is a good option for translating *this* component only but it is not suitable for translating child components which require only the 2digit option (here: react-timeago). Unless i18next can also provide the translation (3<->2) and I missed it.